### DISCUSSION TITLE
I want to stream to a playlist

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6345] [General]
I want to make a 24/7 playlist loop , but i need to be able to stream to the same channel LIVe too , how is this possible ?

**Comment by yashtandon113** [comment]
@brixxnatt 
There is no direct method to start a playlist and live on the same stream ID on the server side. We will get back to you with some workaround solutions to switch between live and playlist.

**Reply by brixxnatt** [reply]
yes please , what is the work arround

**Reply by brixxnatt** [reply]
any ideas plz

**Reply by mekya** [reply]
Hi @yashtandon113,

Reminder for you to give the workaround solution.

Cheers
Ozu

**Comment by yashtandon113** [comment]
HI @brixxnatt 

Sorry for the delay in response, as I had missed it.

The workaround solution that I was going to provide you was using the live m3u8 and VOD in the same playlist so that if the live goes out, the playlist will switch to the VOD for the same stream ID.

But due to some issues, it did not work for me. We will check and get back to you.

**Comment by yashtandon113** [comment]
Hi @brixxnatt 

We have found a workaround solution. You can follow the instructions in the following Github discussions.

https://github.com/orgs/ant-media/discussions/6698

If you have any questions, please feel free to ask.
